dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/preheat/supplier/EventProgramStageMapSupplier.java

```diff
@@ -81,9 +81,10 @@ public void preheatAdd(TrackerObjects trackerObjects, TrackerPreheat preheat) {
 
     List<String> notRepeatableProgramStageUids =
         trackerObjects.getEvents().stream()
-            .map(Event::getProgramStage)
+            .map(Event::getProgram)
```

Comment:
Why do you need to change this code?
We are potentially introducing a lot of unnecessary data into the Preheat.